src/CatalogService/Catalog.Api/Controllers/DocumentController.cs

```diff
@@ -20,6 +23,19 @@ public class DocumentController : ApiControllerBase
         
         _catalogsIndexName = Configuration[$"{environment}ElasticConfig:CatalogsIndexName"];
         _categoriesIndexName = Configuration[$"{environment}ElasticConfig:CategoriesIndexName"];
+        _reviewsIndexName = Configuration[$"{environment}ElasticConfig:ReviewsIndexName"];
+    }
```

Comment:
extract all indexNames to the base class to make derived controllers cleaner and check it through all controllers